pages/editAccount.tsx

```diff
@@ -33,10 +34,13 @@ export default function EditAccount() {
     const fetchUsers = async () => {
         const userList = await UserService.getSelfUser();
         setUsers(userList);  
-        console.log(userList);
-        console.log(userList.name); 
-
+        // console.log(userList);
+        // console.log(userList.name); 
+         //console.log(typeof userList.name);
+         name1 = userList.name;
```

Comment:
You can simply call`setName(user.name)` when variable `name` is already a state.
Then the value in the input HTML will change automatically.